lib/anycablebility/cli/app.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'anycablebility/rpc'
+require 'anycablebility/tests'
+require 'anycablebility/client_factory'
+
+module Anycablebility
+  module Cli
+    # This class is the entry point of Anycablebility
+    class App
+      def run(
+        target: 'ws://0.0.0.0:8080/cable',
+        redis: 'redis://localhost:6379',
+        debug: false
+      )
+        logger = Logger.new(STDOUT)
+        logger.level = debug ? Logger::DEBUG : Logger::WARN
+
+        rpc = Rpc.instance.configure(redis, debug, logger)
```

Comment:
Done.

Could you please take a look at how I require stuff? I'm not sure I do it the right way. I have PHP background so I require components where do I need them, like PHP programmers `use` their classes and constants. Is that right paradigm? Or we should make some kind of tree with `require`s like [rubocop does](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop.rb)?